### DISCUSSION TITLE
program cil_xdp_entry: attaching XDP program to interface ens33: operation not supported

### DISCUSSION THREAD

**Discussion by Einic** [cilium/ebpf#1185] [Q&A]
![image](https://github.com/cilium/ebpf/assets/50362911/07a52348-d3fd-41f4-a027-ebe5283a5247)
kernel version: 6.0.0-1.el7.elrepo.x86_64
cilium version: v1.14.3	
kubernetes: v1.28.0
CentOS Linux release 7.9.2009 (Core)	
The current environment is deployed and runs in VMware® Workstation 17 Pro centos linux environment，How to enable XDP program operation supported here ?

**Comment by lmb** [comment]
Hi, seems like your problem is related to cilium itself, not the ebpf library. Your best bet is to try on slack: https://cilium.io/get-help/
